sdk/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -296,7 +306,6 @@ public void setIgnoreClasses(String[] ignoreClasses) {
      */
     public void setNotifyReleaseStages(@Nullable String[] notifyReleaseStages) {
         this.notifyReleaseStages = notifyReleaseStages;
-        notifyBugsnagObservers(NotifyType.RELEASE_STAGES);
```

Comment:
this one seems to have been removed and not replaced with anything, is that because it is only needed by the java part of the notifier?

Reply:
Correct! The delivery/release stage handling now only occurs on the Java side.